views/helpers/index.js

```diff
@@ -126,6 +128,16 @@ function checkResultsFromList(array, scenario) {
   }
 }
 
+
+var ifEquals = function(a, b, opts) {
+  if (a == b) {
```

Comment:
`==` will use type coercion if the types don't match and that can make weird things happens, so generally in JS `===` is used unless you want things like `0 == false // true`